s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -44,6 +35,18 @@ export class MapComponent implements OnInit, OnDestroy {
   constructor(private CONFIG: S4eConfig) {
   }
 
+  private _overlays: UIOverlay[] = [];
+
+  get overlays(): UIOverlay[] {
+    return this._overlays;
+  }
+
+  @Input() set overlays(value: UIOverlay[]) {
+    console.log(this._overlays);
```

Comment:
console logs should not usually go to the master (I know, that it's not always a rule...)